Client/Frontend/Browser/URLBarView.swift

```diff
@@ -637,11 +637,11 @@ private class CurveView: UIView {
 private class ToolbarTextField: AutocompleteTextField {
     override func textRectForBounds(bounds: CGRect) -> CGRect {
         let rect = super.textRectForBounds(bounds)
-        return rect.rectByInsetting(dx: 5, dy: 5)
+        return rect.rectByInsetting(dx: 10, dy: 5)
```

Comment:
Don't know where the original 5 came from - was hard-coded in there - but the increase was to align it to the Label in BrowserLocationView. I tested on several different devices and the offset of 10 for placement of the text aligns exactly with the placement of the constraints on the BrowserLocationView Label.